CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # FL Accelerator Changelog
 
+##  October 2022 release
+
+We are excited to announce the release of the September iteration of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning).
+
+Here are the new features.
+
+
+### Repository structure
+- Enabled CI/CD on the repo to ensure nothing breaks.
+
+### Provisioning
+- Decoupled the provisioning of the workspace, the orchestrator, and the silos.
+- New scripts for provisioning various kinds of resources, all listed in the [cookbook](./docs/provisioning/README.md) (orchestrator behind vnet, silos behind vnets, silos using an AKS cluster with confidential compute, silos using an existing storage account).
+- Introduced "one-click deployments" for all resources.
+
+### Documentation
+- Instructions on how to provision [external silos](./docs/provisioning/external-silos.md).
+- Tutorial on [how to adapt literal and factory code to your scenarios](./docs/literal-factory-tutorial.md).
+
+### FL Experience
+- Added training subgraphs to the "factory" example for a better user experience.
```

Comment:
proposed rewording:

- Using AzureML SDK subgraphs (preview) as basis for the factory code sample, so that writing an FL experiment is easier, more flexible, and showing better in the Azure ML UI.